pepper-apis/src/main/resources/logback.xml

```diff
@@ -20,13 +20,12 @@
         <filter class="ch.qos.logback.classic.filter.LevelFilter">
             <level>ERROR</level>
             <OnMismatch>DENY</OnMismatch>
-            <OnMatch>ALLOW</OnMatch>
+            <OnMatch>ACCEPT</OnMatch>
         </filter>
     </appender>
 
     <logger name="org.redisson" level="info">
         <appender-ref ref="STDOUT" />
-        <appender-ref ref="SLACK" />
```

Comment:
No more Slacking of Redisson logs.

Reply:
Nice!  These will continue to show up in GCP error logging, where we can decide later whether to mute or not on-the-fly.